test/lit/d8/fuzz_shell_jspi.wast

```diff
@@ -0,0 +1,78 @@
+(module
+  (import "fuzzing-support" "log-i32" (func $log (param i32)))
+
+  (func $a (export "a") (result i32)
+    (i32.const 10)
+  )
+
+  (func $b (export "b") (result i32)
+    (i32.const 20)
+  )
+
+  (func $c (export "c") (result i32)
+    (i32.const 30)
+  )
+
+  (func $d (export "d") (result i32)
+    (i32.const 40)
+  )
+
+  (func $e (export "e") (result i32)
+    (i32.const 50)
+  )
+)
+
+;; Apply JSPI: prepend JSPI = 1 and remove comments around async and await.
+
+;; RUN: echo "JSPI = 1;" > %t.js
+
+;; Use |echo| here to avoid quoting issues on windows paths.
+;; RUN: echo %S | node -e "process.stdout.write(require('fs').readFileSync(path.join(require('fs').readFileSync(0, 'utf-8'), '..', '..', '..', 'scripts', 'fuzz_shell.js'), 'utf-8').replace(/[/][*] async [*][/]/g, 'async').replace(/[/][*] await [*][/]/g, 'await'))" >> %t.js
```

Comment:
Can this go in a helper script of some kind? At the very least, could we split the logic across multiple RUN lines?

Reply:
heh, yeah, this is a pretty horrible line. It was a hard-fought battle to even get it to work across all platforms!

I did some more work to simplify it now, it turns out `lit` has `cat` support (at least on linux... we'll see on CI for windows). If that works, I don't think we can split any more.